Description: work-around for test failure due to parent class being
 called for each setUp().
Author: tony mancill <tmancill@debian.org>

--- a/test/bitronix/tm/mock/JmsProperUsageMockTest.java
+++ b/test/bitronix/tm/mock/JmsProperUsageMockTest.java
@@ -101,9 +101,11 @@
         assertEquals(Status.STATUS_COMMITTING, ((JournalLogEvent) orderedEvents.get(i++)).getStatus());
         assertEquals(true, ((XAResourceCommitEvent) orderedEvents.get(i++)).isOnePhase());
         assertEquals(Status.STATUS_COMMITTED, ((JournalLogEvent) orderedEvents.get(i++)).getStatus());
-    }
 
-    public void testSerialization() throws Exception {
+	// setUp/tearDown cannot be called twice
+    //}
+
+    //public void testSerialization() throws Exception {
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         ObjectOutputStream oos = new ObjectOutputStream(baos);
         oos.writeObject(poolingConnectionFactory1);
